int/api/websites/webcreator.go

```diff
@@ -28,6 +30,25 @@ func CreatePrepareForWebsiteHandler(
 	}
 }
 
+func listArchiveContent(zipReader *zip.Reader) []string {
+	FilesInArchive := []string{}
+	for _, zipFile := range zipReader.File {
+		FilesInArchive = append(FilesInArchive, zipFile.Name)
+	}
+
+	return FilesInArchive
+}
+
+func contains(elems []string, v string) bool {
+	for _, s := range elems {
+		if v == s {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
The best code is the one you're not writing : https://pkg.go.dev/golang.org/x/exp/slices#Index